src/ipahealthcheck/ipa/config.py

```diff
@@ -44,3 +51,56 @@ def check(self):
                 constants.SUCCESS,
                 key='krbLastSuccessfulAuth'
             )
+
+
+class SSSDAllowedUids389Check(IPAPlugin):
+    """
+    Checks if UID 389 (LDAP service account) is listed in allowed_uids
+    in sssd.conf.
+
+    If UID 389 is in allowed_uids, SSSD will prevent local resolution
+    which will cause issues with the IPA services.
+    """
+
+    @duration
+    def check(self):
+        try:
+            sssdconfig = SSSDConfig.SSSDConfig()
+            sssdconfig.import_config()
+        except Exception as e:
+            logger.debug('Failed to parse sssd.conf: %s', e)
+            yield Result(self, constants.CRITICAL, error=str(e),
+                         msg='Unable to parse sssd.conf: {error}')
+            return
+
+        try:
+            service = sssdconfig.get_service('pac')
+        except SSSDConfig.NoServiceError:
+            yield Result(
+                    self, constants.ERROR,
+                    key='SSSD_allowed_uids',
+                    msg="No [pac] section defined in sssd.conf"
+            )
+            return
```

Comment:
I think we should allow this case and treat it the same as an empty `[pac]` section.  SSSD runs fine when the section is not defined.